src/app/portlets/content-types/edit/content-types-edit.component.ts

```diff
@@ -73,4 +76,25 @@ export class ContentTypesEditComponent extends BaseComponent {
         this.form.resetForm();
         this.readyToAddFields = true;
     }
+
+    private deleteContentType(id): void {
+        this.crudService.delete(`v1/contenttype/id/`, id)
+            .subscribe((data) => {
+                this.router.navigate(['../'], { relativeTo: this.route });
+            });
+    }
+
+    private deleteContentTypeDialog($event): any {
```

Comment:
This should be named: ```deleteContentType``` and in the accept you do the request, because that the action the button it's triggering, if in the future we remove the dialog we shouldn't change the name of the method.

If you want to move the request itself to another method that's ok too.